account_due_list/account_move_line.py

```diff
@@ -45,6 +45,66 @@ class AccountMoveLine(models.Model):
                                         compute='_get_invoice',
                                         string='Invoice', store=True)
 
+    maturity_residual = fields.Float(
+        compute='_maturity_residual', string="Residual Amount", store=True,
+        help="The residual amount on a receivable or payable of a journal "
+             "entry expressed in the company currency.")
+
+    @api.multi
+    @api.depends('date_maturity')
```

Comment:
It seems to me it should also depend on other fields, like reconcile_id, account_id.reconcile, amount_currency, debit, credit, reconcile_partial_id.line_partial_ids, currency_id, company_id.currency_id